### ISSUE TITLE
Feedback Submission - tutorials - Build a proof of existence dApp - /tutorials/v3/proof-of-existence

### ISSUE THREAD

**Issue by eum602** [substrate-developer-hub/substrate-docs#627]
### What browsers are you seeing the problem on?

Other (please write below)

### Feedback

In the "Proof of existence" tutorial **the following line of code should be omitted** since this generates an error during compilation:
```shell
#[pallet::generate_storage_info]
```
The error is:
```shell
error[E0277]: the trait bound `Vec<u8>: MaxEncodedLen` is not satisfied
  --> pallets/template/src/lib.rs:40:11
   |
40 | #[pallet::generate_storage_info]
   |           ^^^^^^^^^^^^^^^^^^^^^ the trait `MaxEncodedLen` is not implemented for `Vec<u8>`
   |
   = note: required because of the requirements on the impl of `StorageInfoTrait` for `frame_support::pallet_prelude::StorageMap<_GeneratedPrefixForStorageProofs<T>, frame_support::Blake2_128Concat, Vec<u8>, (<T as frame_system::Config>::AccountId, <T as frame_system::Config>::BlockNumber), frame_support::pallet_prelude::ValueQuery>`
note: required by `storage_info`
  --> /home/acc-1/.cargo/git/checkouts/substrate-7e08433d4c370a21/bf9683e/frame/support/src/traits/storage.rs:71:2
   |
71 |     fn storage_info() -> Vec<StorageInfo>;
```

**Comment by NukeManDan** [comment]
Same reported here: 
https://stackoverflow.com/questions/70206199/substrate-tutorials-trait-maxencodedlen-is-not-implemented-for-vecu8#comment124106274_70206199

**Comment by NukeManDan** [comment]
I believe all that is needed it to _remove_ the `#[pallet::generate_storage_info]` macro line. @sacha-l  this right?
IIRC, this was fixed previously.... Perhaps we can fix this and more in #648
